internal/config/config.go

```diff
@@ -188,30 +188,31 @@ type Config struct {
-	Debug                           bool   `envconfig:"DEBUG" default:"false"`
-	Trace                           bool   `envconfig:"TRACE" default:"false"`
-	DisableSecretCreation           bool   `envconfig:"DISABLE_SECRET_CREATION" default:"false"`
-	TestkubeExecutionNamespaces     string `envconfig:"TESTKUBE_EXECUTION_NAMESPACES" default:""`
-	GlobalWorkflowTemplateName      string `envconfig:"TESTKUBE_GLOBAL_WORKFLOW_TEMPLATE_NAME" default:""`
-	GlobalWorkflowTemplateInline    string `envconfig:"TESTKUBE_GLOBAL_WORKFLOW_TEMPLATE_INLINE" default:""`
-	EnableK8sEvents                 bool   `envconfig:"ENABLE_K8S_EVENTS" default:"true"`
-	TestkubeDockerImageVersion      string `envconfig:"TESTKUBE_DOCKER_IMAGE_VERSION" default:""`
-	DisableDeprecatedTests          bool   `envconfig:"DISABLE_DEPRECATED_TESTS" default:"false"`
-	DisableWebhooks                 bool   `envconfig:"DISABLE_WEBHOOKS" default:"false"`
+	DisableDefaultAgent             bool     `envconfig:"DISABLE_DEFAULT_AGENT" default:"false"`
+	TestkubeConfigDir               string   `envconfig:"TESTKUBE_CONFIG_DIR" default:"config"`
+	TestkubeAnalyticsEnabled        bool     `envconfig:"TESTKUBE_ANALYTICS_ENABLED" default:"false"`
+	TestkubeNamespace               string   `envconfig:"TESTKUBE_NAMESPACE" default:"testkube"`
+	TestkubeProWorkerCount          int      `envconfig:"TESTKUBE_PRO_WORKER_COUNT" default:"50"`
+	TestkubeProLogStreamWorkerCount int      `envconfig:"TESTKUBE_PRO_LOG_STREAM_WORKER_COUNT" default:"25"`
+	TestkubeProMigrate              string   `envconfig:"TESTKUBE_PRO_MIGRATE" default:"false"`
+	TestkubeProRunnerCustomCASecret string   `envconfig:"TESTKUBE_PRO_RUNNER_CUSTOM_CA_SECRET" default:""`
+	CDEventsTarget                  string   `envconfig:"CDEVENTS_TARGET" default:""`
+	TestkubeDashboardURI            string   `envconfig:"TESTKUBE_DASHBOARD_URI" default:""`
+	TestkubeClusterName             string   `envconfig:"TESTKUBE_CLUSTER_NAME" default:""`
+	TestkubeHelmchartVersion        string   `envconfig:"TESTKUBE_HELMCHART_VERSION" default:""`
+	DebugListenAddr                 string   `envconfig:"DEBUG_LISTEN_ADDR" default:"0.0.0.0:1337"`
+	EnableDebugServer               bool     `envconfig:"ENABLE_DEBUG_SERVER" default:"false"`
+	Debug                           bool     `envconfig:"DEBUG" default:"false"`
+	Trace                           bool     `envconfig:"TRACE" default:"false"`
+	DisableSecretCreation           bool     `envconfig:"DISABLE_SECRET_CREATION" default:"false"`
+	TestkubeExecutionNamespaces     string   `envconfig:"TESTKUBE_EXECUTION_NAMESPACES" default:""`
+	GlobalWorkflowTemplateName      string   `envconfig:"TESTKUBE_GLOBAL_WORKFLOW_TEMPLATE_NAME" default:""`
+	GlobalWorkflowTemplateInline    string   `envconfig:"TESTKUBE_GLOBAL_WORKFLOW_TEMPLATE_INLINE" default:""`
+	TransferEnvVariables            []string `envconfig:"TRANSFER_ENV_VARS" default:"GRPC_ENFORCE_ALPN_ENABLED"`
```

Comment:
Should we keep it as `default` here, as users will need to remember to add it, in case they define `TRANSFER_ENV_VARS ` for some other variable?

Reply:
It depends what we want to achieve. I believe it will be used only for workarounds. In any case (1) we can enforce it on Helm Chart, (2) this env variable will be barely needed, (3) it's deprecated and soon deleted.